llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -1455,21 +1513,51 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     // correct origin.
     bool Combine = !MS.TrackOrigins;
     Instruction *Instruction = InstructionChecks.front().OrigIns;
+
+    Value *InstName = nullptr;
+    if (ClEmbedFaultingInst >= 1) {
+      IRBuilder<> IRB0(Instruction);
+      std::string str;
```

Comment:
I think this is only used in one branch?

Reply:
Moved